src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -2086,7 +2086,7 @@ public sealed class NoArgumentsInput : CalibrateInputBase
         public sealed class FixedPlattInput : CalibrateInputBase
         {
             [Argument(ArgumentType.AtMostOnce, ShortName = "slope", HelpText = "The slope parameter of the calibration function 1 / (1 + exp(-slope * x + offset)", SortOrder = 1)]
```

Comment:
>-slope [](start = 141, length = 6)

Can you change this to `exp(slope * x + offset)`? (and in the `Offset` argument as well).